cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -410,6 +418,11 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 	).Run()
 	time.Sleep(wait.Jitter(s.ControllerStartInterval.Duration, ControllerStartJitter))
 
+	// run the shared informers
+	for _, informer := range informers {
+		go informer.Run(wait.NeverStop)
```

Comment:
@deads2k I'm creating a new controller so I was looking into using a shared informer. Some questions:

1. Are shared informers ready for wider use, or are they still a work in progress? 
2. All shared informers (including the `podInformer`) are started here (`Run(...)` is called) in the `controllermanager`, but I see that the individual controllers also start the pod informer passed into them manually (`internalPodInformer.Run(...)`), won't this result in the `podInformer`'s `Run(...)` getting called twice?